roles/dotfiles/templates/zshrc.j2

```diff
@@ -40,8 +40,6 @@ export PATH="/usr/local/opt/python/libexec/bin:$PATH"
 # gpg and password prompt
 export GPG_TTY=$(tty)
 
-export REQUESTS_CA_BUNDLE="/usr/local/etc/openssl/cacert.pem"
```

Comment:
without this line, I cannot execute the infra playbooks

Reply:
It's back.